tests/product_tests/foreman/foreman_tests.py

```diff
@@ -0,0 +1,87 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+
+from sos_tests import StageOneReportTest, redhat_only
+
+
+class ForemanBasicTest(StageOneReportTest):
+    """Ensure that a basic execution runs as expected with all TFM related
+    plugins. It assumes Foreman runs on a Katello deployment.
+
+    :avocado: tags=foreman
+    """
+
+    sos_cmd = '-v'
+
+    def test_tfm_plugins_ran(self):
+        self.assertPluginIncluded([
+            'apache',
+            'foreman',
+            'candlepin',
+            'postgresql',
+            'puppet',
+            'ruby'
+        ])
+
+    def test_foreman_keys_skipped(self):
+        self.assertFileGlobNotInArchive("/etc/foreman*/*key.pem")
+
+    def test_foreman_database_sizes_collected(self):
+        self.assertFileCollected('sos_commands/foreman/foreman_db_tables_sizes')
+
+    def test_foreman_installer_dirs_collected(self):
+        self.assertFileGlobInArchive("/etc/foreman-installer/*")
+        self.assertFileGlobInArchive("/var/log/foreman-installer/*")
+
+    def test_foreman_production_log_collected(self):
+        self.assertFileCollected('/var/log/foreman/production.log')
+
+    def test_foreman_database_dumps_collected(self):
+        self.assertFileCollected('sos_commands/foreman/foreman_settings_table')
+
+    def test_foreman_csv_collected(self):
+        self.assertFileCollected('sos_commands/foreman/foreman_tasks_tasks')
+
+    def test_proxyfeatures_not_collected(self):
+        self.assertFileGlobNotInArchive("*/smart_proxies_features/*")
+
+    def test_candlepin_table_sizes_collected(self):
+        self.assertFileCollected('sos_commands/candlepin/candlepin_db_tables_sizes')
+
+    def test_foreman_password_postproc_worked(self):
+        # known value for our sealed test image
+        self.assertFileNotHasContent(
+            '/var/log/foreman-installer/katello.log',
+            'rs74B8sAiaEJtDzdVgu5SAP2TZHQSDS4'
```

Comment:
+1 for having a variable for the password (can't see the Ewoud's comment on this).